### ISSUE TITLE
Dupe with crafting glow frames

### ISSUE THREAD

**Issue by SlimeCody** [tiffany352/InvisibleItemFrames#2]
https://user-images.githubusercontent.com/49579480/125347173-399d4980-e320-11eb-9842-475ad69d0cb7.mp4

**Comment by SlimeCody** [comment]
NVM resolved by updating the configuration

**Comment by tiffany352** [comment]
Yep, sorry about that. The config auto-conversion is a bit broken, it can be fixed by manually updating the config.

I'll try to fix this next time I work on the plugin.
